.travis.yml

```diff
@@ -1,9 +1,13 @@
 language: ruby
-dist: trusty
 cache: bundler
+sudo: required
+
+rvm:
+- 2.5.1
+- 2.6.1
```

Comment:
These changes to the `.travis.yml` look unrelated to the rest of this PR (updating the vcp api version).

Can you split it out into a separate PR?